### PR TITLE
Update bundled root CA certs, improve root CA cert management

### PR DESCRIPTION
**Pull request by Kami**
This pull request includes the following changes:

* Updates default value for the ``ssl_ca_bundle_path`` config option to ``/etc/ssl/certs/ca-certificates.crt``. Previous default value was wrong and the reason why the agent didn't correctly pick up system CA bundle on Debian based distros (including default OSS logstash images).
* Update root ca certs we bundle with the plugin. Previously we only bundled section root ca cert, now we bundle all the root certs used by DataSet endpoints (same root certs as we bundle with the agent - https://github.com/scalyr/scalyr-agent-2/tree/master/certs)
* ~~In case ``append_builtin_cert`` config option is false, we don't copy file specified as part of ``ssl_ca_bundle_path`` since there is not need to - to account for this change, some tests needed to be updated.~~

## TODO

- [x] Update affected tests
- [ ] Improve test messages and errors which are thrown if specified ca path doesn't exist - we should throw early instead of letting it go all the way to Manticore and throw there in case empty CA path is used - that seems fragile and dangerous
- [x] Add documentation on those config options and explain how cert validation works

### CODE REVIEW

**Comment by Kami** [comment]
NOTE: I will update the code to throw an explicit error message early on in case ``ssl_ca_bundle_path`` config option is specified but it points to an invalid / nonexistent path.

Previously an empty CA certs file would get created and this value will get passed all the way down to Manticore HTTP client which would then throw ``error: java.security.InvalidAlgorithmParameterException: the trustAnchors parameter must be non-empty`` error.

imo, that behavior is not error prone and dangerous - in case Manticore behavior ever changes to not perform SSL validation or similar in case that file is empty instead of throwing.

**Comment by Kami** [comment]
Re my comment above^ - turns out I can't do that if I want to keep the change fully backward compatible :/

So for now, I will take that change out and add it back in a future release when we further improve and clean up that code.

**Review by ArthurKamalov — APPROVED** [review]
Looks good, so far.
Sorry for the post-merge review :D

File: scripts/concatenate_root_certs.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
+#
```
Comment (ArthurKamalov) [inline scripts/concatenate_root_certs.py:1]: It seems that this script has to be used manually to update the bundle when any of cert files are changed, right? Maybe GHA can do that?

**Review by ArthurKamalov — COMMENTED** [review]
Still LGTM

File: scripts/concatenate_root_certs.py (hunk 1 above)
Comment (Kami) [reply scripts/concatenate_root_certs.py:1]: The problem is that the change needs to be committed - so we would need to have workflow also commit and push to the repo.

**Pull request merged**